miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -474,6 +504,7 @@ internal object ErrorBridgeMessage {
     const val NO_IMPLEMENT_CUSTOM_PERMISSION =
         "The `MiniAppMessageBridge.requestCustomPermissions` $NO_IMPL"
     const val ERR_SHARE_CONTENT = "Cannot share content:"
+    const val ERR_UNIVERSAL_BRIDGE = "Please check the String and try again."
```

Comment:
I think, we'll have real error message when there will be exception, so, this variable can be used as a prefix to represent the feature.

```suggestion
    const val ERR_UNIVERSAL_BRIDGE = "Cannot send json info:"
```